src/components/RadioGroupField/RadioGroupField.tsx

```diff
@@ -50,7 +50,8 @@ interface RadioGroupFieldProps {
   disabled?: boolean;
   error?: boolean;
   hint?: string;
-  label?: string;
+  label?: React.ReactNode;
+  description?: string;
```

Comment:
Wouldn't `React.ReactNode` be better?

Reply:
Oh, description shouldn't be prop anymore, I forgot to remove it. Removed.